repos/system_upgrade/el7toel8/actors/selinux/selinuxcontentscanner/actor.py

```diff
@@ -0,0 +1,193 @@
+import os
+import re
+from shutil import rmtree
+import subprocess
```

Comment:
the `subprocess` library should be omitted in favor of `leapp.libraries.stdlib` functions. When you find any use that you are not able to do with proposed functions, let me know and we will try to implement that. for each use of subprocess that is necessary, add please `# FIXME` line, so we can discuss it.